pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -748,12 +749,14 @@ func resolveBackupManifests(
 						const errPrefix = "invalid RESTORE timestamp: restoring to arbitrary time requires that BACKUP for requested time be created with '%s' option."
 						if i == 0 {
 							return nil, nil, nil, errors.Errorf(
-								errPrefix+" nearest backup time is %s", backupOptRevisionHistory, b.EndTime,
+								errPrefix+" nearest backup time is %s", backupOptRevisionHistory,
+								time.Unix(0, b.EndTime.WallTime).UTC(),
 							)
 						}
 						return nil, nil, nil, errors.Errorf(
 							errPrefix+" nearest BACKUP times are %s or %s",
-							backupOptRevisionHistory, mainBackupManifests[i-1].EndTime, b.EndTime,
+							backupOptRevisionHistory, time.Unix(0, mainBackupManifests[i-1].EndTime.WallTime),
```

Comment:
Looks like the first time isn't in UTC?

Reply:
oops, nice catch, fixed and switched to using timeutil since lint insists.